buildSrc/src/main/kotlin/godot/codegen/Class.kt

```diff
@@ -222,8 +224,8 @@ class Class @JsonCreator constructor(
             typeBuilder.addInitializerBlock(
                 CodeBlock.of("""
                             |%M {
-                            |    val ptr = %M(%T.gdnative.godot_global_get_singleton).%M("$oldName".%M.ptr)
-                            |    %M(ptr) { "No instance found for singleton $newName" }
+                            |    val ptr = %M(%T.gdnative.godot_global_get_singleton).%M("$singletonName".%M.ptr)
+                            |    %M(ptr) { "No instance found for singleton $singletonName" }
```

Comment:
I would continue to use newName in log, so that it is easier to retrieve from where comes the error.

Reply:
It's probably harder to debug though, as the newName != singletonName - you have to look up at the source to know what name are we exactly passing.